spec/features/person_displays_spec.rb

```diff
@@ -18,126 +19,100 @@
       create(:person, firstname: 'Donna', status: 'Declined' )
-      create(:person, firstname: 'Oscar', status: 'Active',  department: dpw)
+      create(:person, firstname: 'Oscar', status: 'Active',  department: @dpw)
+      @people_manage_depts = Department.where(manage_people: true)
+      @all_depts = Department.all
     end
 
-    it "returns the index page" do
+    it "returns page for All Active people", js: true do
       visit people_path
-      expect(page).to have_content('People')
+      # Active checked by default
+      expect(page).to have_content('People') # This is in the nav bar
+      expect(page).to have_content('All Active')
+      expect(page).to have_content('Active:')
+      expect(page).to have_content('On-leave:')
+      expect(page).to have_checked_field('Active')
+      expect(page).to have_unchecked_field('On-leave')
+      within(".data-table-departments") do
+        expect(page).to have_content('Departments:')
+        @people_manage_depts.each do |dept|
+          expect(page).to have_content(dept.shortname)
+        end
+      end
+      expect(page).not_to have_content(@dpw.shortname)
       expect(page).to have_content('Home') # This is in the nav bar
       expect(page).to have_content('CJ')
-      expect(page).to have_content('CERT')
-      expect(page).to have_content('Sierra')
-      expect(page).to_not have_content('Adam') # Should not show applicant
-      expect(page).to_not have_content('Priscilla')
-      expect(page).to_not have_content('Indy')
-      expect(page).to_not have_content('Leona')
-      expect(page).to_not have_content('Oscar')
-    end
-
-    it "returns a page for Police" do
-      visit department_people_path(Department.find_by(shortname: "BAUX").id)
-      expect(page).to have_content('Police')
-      expect(page).to have_content('Home') # This is in the nav bar
-      expect(page).to have_content('CJ')
-      expect(page).not_to have_content('Sierra')
+      expect(page).to have_content('Sierra') # Should only show active people from depts managed by people
       expect(page).not_to have_content('Adam')
       expect(page).not_to have_content('Priscilla')
       expect(page).not_to have_content('Indy')
       expect(page).not_to have_content('Leona')
       expect(page).not_to have_content('Oscar')
-   end
+      page.has_css?('add-btn')
 
-    it "returns a page for CERT" do
-      visit department_people_path(Department.find_by(shortname: "CERT").id)
-      expect(page).to have_content('CERT')
-      expect(page).to have_content('Home') # This is in the nav bar
-      expect(page).not_to have_content('CJ')
-      expect(page).to have_content('Sierra')
-      expect(page).not_to have_content('Adam')
-      expect(page).not_to have_content('Priscilla')
-      expect(page).not_to have_content('Indy')
-      expect(page).not_to have_content('Oscar')
-      expect(page).not_to have_content('Leona')
+      check "js-people-onLeave-checkbox"
```

Comment:
Really nice. 
Could you do `expect(page_.to_not have_content('Leona')` right before the `check` ?